jwst/lib/set_telescope_pointing.py

```diff
@@ -22,6 +26,36 @@
 # Setup logging
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.NullHandler())
+DEBUG_FULL = logging.DEBUG - 1
+LOGLEVELS = [logging.INFO, logging.DEBUG, DEBUG_FULL]
+
+
+# The available methods for transformation
+class Methods(Enum):
+    GSCMD_J3PAGS = ('gscmd', 'calc_transforms_gscmd_j3pags')         # JSOCINT-555 fix using J3PA@GS
+    GSCMD_V3PAGS = ('gscmd_v3pags', 'calc_transforms_gscmd_v3pags')  # JSOCINT-555 fix using V3PA@GS
+    ORIGINAL = ('original', 'calc_transforms_original')              # Original, pre-JSOCINT-555 algorithm
+
+    # Alias
+    default = ORIGINAL    # Use original algorithm if not specified
+    GSCMD = GSCMD_J3PAGS  # When specifying GS Commanded, use the J3VA@GS method by default.
+
+    def __new__(cls: object, value: str, func_name: str):
+        obj = object.__new__(cls)
+        obj._value_ = value
+        obj._func_name = func_name
+        return obj
+
+    @property
+    def func(self):
+        return globals()[self._func_name]
+
+    def __str__(self):
+        return self.value
+
+
+# Definition of th J3 Idle Y-Angle
```

Comment:
typo? Shouldn't "Idle" be "Ideal"?  (i.e. in the Ideal frame)

Reply:
✅